### DISCUSSION TITLE
Differently coloured from and import statements in Python

### DISCUSSION THREAD

**Discussion by isaksamsten** [olimorris/onedarkpro.nvim#177] [Ideas]
Is this intentional?

`["@include.python"] = { fg = theme.palette.fg, style = config.styles.keywords },`
[L12](https://github.com/olimorris/onedarkpro.nvim/blob/c4a4a1120c89665b2e4bfd4bc7dcbd96dd87b1ee/lua/onedarkpro/highlights/filetypes/python.lua#L12) 

and

`["@odp.import_from.python"] = { fg = theme.palette.purple, style = config.styles.keywords },` [L18](https://github.com/olimorris/onedarkpro.nvim/blob/c4a4a1120c89665b2e4bfd4bc7dcbd96dd87b1ee/lua/onedarkpro/highlights/filetypes/python.lua#L18) 

Perhaps both should be purple? I’ve overridden the highlight to have them both purple so no real fix is needed, but if if this is unintentional consider this a bug report :-)

Thanks for creating the best colour scheme!

**Comment by olimorris** [comment]
Yep. As per the readme, this is based on the Visual Studio Code theme. So try to get as close to that as possible.

Sometimes we need custom Treesitter queries to get additional highlights.

**Reply by isaksamsten** [reply]
Thanks for the reply! I was using theme in VSCode but dont recall the difference, but i might have overriden it and forgot about it.

**Reply by olimorris** [reply]
Btw, if I have got this wrong. Do send over a screenshot from VSCode and I'll gladly amend.

**Reply by isaksamsten** [reply]
<img width="1092" alt="Screenshot 2023-04-02 at 21 10 57" src="https://user-images.githubusercontent.com/167128/229373830-2a1416f5-8749-4c32-b33f-15c4d8faee71.png">

**Comment by olimorris** [comment]
This is now fixed as per 455dfbe87678d7c91a0f67edb7c7aaac54539fa5
